src/main/kotlin/org/sertain/path/PathController.kt

```diff
@@ -0,0 +1,17 @@
+package org.sertain.path
+
+/**
+ * Created by jacksoncoder on 1/10/18.
+ */
+
+object PathController {
+    val pathCoordinates = ArrayList<Triple<Float,Float,Float>>()
```

Comment:
Space after commas (IntelliJ should take care of this if you code format I believe).